src/instructlab/lab.py

```diff
@@ -973,6 +973,16 @@ def convert(self, value, param, ctx) -> "torch.device":
     show_default=True,
     help="model name to use in training",
 )
+@click.option(
+    "--override-training-args",
+    default="{}",
+    show_default=True,
+    hidden=True,
+    help=(
+        "Additional arguments for training (json string)"
+        "e.g. '--override-training-args {\"gradient_accumulation_steps\"=8}'"
+    ),
+)
```

Comment:
I think the main blocker now for this PR is getting this reviewed/merged separately.

Reply:
I know you have #1008 for this.

There's a design doc that's in this space as well: https://github.com/instructlab/dev-docs/pull/52